codegen/template.go

```diff
@@ -60,7 +60,7 @@ var defaultFuncMap = tmpl.FuncMap{
 	"dec":           decrement,
 	"basePath":      filepath.Base,
 	"pascal":        PascalCase,
-	"jsonMarshal":   jsonMarshal,
+	"yamlMarshal":   yamlMarshal,
```

Comment:
nit: from what I could tell this function is unused in the template.

Reply:
function removed